src/debug/debug.cc

```diff
@@ -1229,17 +1230,17 @@ debugLogTime(const time_t t)
         // 4 bytes smaller than buf to ensure .NNN catenation by snprintf()
         // is safe and works even if strftime() fills its buffer.
         char buf2[sizeof(buf)-4];
-        tm = localtime(&t);
+        tm = localtime(&(t.tv_sec));
```

Comment:
Personally, I find the inner parenthesis excessive, but I do not insist on their removal:

```suggestion
        tm = localtime(&t.tv_sec);
```

There is one more such use case below.